packages/core/src/codewhisperer/client/user-service-2.json

```diff
@@ -2,11 +2,14 @@
     "version": "2.0",
     "metadata": {
         "apiVersion": "2022-11-11",
+        "auth": ["smithy.api#httpBearerAuth"],
         "endpointPrefix": "amazoncodewhispererservice",
         "jsonVersion": "1.0",
         "protocol": "json",
+        "protocols": ["json"],
         "serviceFullName": "Amazon CodeWhisperer",
         "serviceId": "CodeWhispererRuntime",
+        "signatureVersion": "bearer",
```

Comment:
not sure do we need to include these 3 lines ornot

Reply:
think it should be fine, confirmed jb has these. https://github.com/aws/aws-toolkit-jetbrains/blob/main/plugins/core/sdk-codegen/codegen-resources/codewhispererruntime/service-2.json#L4-L15

Reply:
Should not include